ETTrace/Tracer/EMGStackTraceRecorder.h

```diff
@@ -0,0 +1,51 @@
+#import <deque>
+#import <vector>
+#import <unordered_map>
+#import <mach/mach.h>
+#import <QuartzCore/QuartzCore.h>
```

Comment:
Minor thing but I think you can remove this import since CACurrentMediaTime is only used in the implementation file

Reply:
We still need something for CFTimeInterval